pkg/sypgp/sypgp.go

```diff
@@ -40,10 +38,7 @@ const helpAuth = `Access token is expired or missing. To update or obtain a toke
 const helpPush = `  4) Push key using "singularity key push %[1]X"
 `
 
-var errPassphraseMismatch = errors.New("passphrases do not match")
-var errTooManyRetries = errors.New("too many retries while getting a passphrase")
 var errNotEncrypted = errors.New("key is not encrypted")
```

Comment:
should this be moved with the other errors?

Reply:
This error is still used in this package within `DecryptKey()` and `RecryptKey()` so it needs to stay